### PR TITLE
* Move CEF context files to local tree

### PR DESCRIPTION
**Pull request by mbonsack**
* Move CEF context files to local tree (as examples)
* Document the files that are now visible to the administrator
* Update all source files for multiple unique ports (list of ports)

### CODE REVIEW

File: tests/test_imperva.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def test_imperva_incapsula(record_property, setup_wordlist, setup_splunk, setup_
 
     sendsingle(message, setup_sc4s[0], setup_sc4s[1][514])
 
-    st = env.from_string("search _time={{ epoch }} index=netwaf host=\"{{ host }}\" sourcetype=\"cef\" source=\"Imperva:Incapsula\"")
+    st = env.from_string("search _time={{ epoch }} index=netwaf host=\"{{ host }}\" sourcetype=\"imperva:incapsula\" source=\"Imperva:Incapsula\"")
```
Comment (rfaircloth-splunk) [inline tests/test_imperva.py:31]: Revert this change the sourcetype is cef on this one

Comment (mbonsack) [reply tests/test_imperva.py]: Reverted...
